src/bootstrap.js

```diff
@@ -81,7 +81,7 @@ module.exports = (ceConfig, pnConfig) => {
     };
 
     ChatEngine.protoPlugins = {};
```

Comment:
Should this be renamed too?

Reply:
the proto stuff is a separate pull

Reply:
but yeah, maybe we can call them protos.